pages/linux/wodim.md

```diff
@@ -0,0 +1,24 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Some usage of wodim can cause destructive actions, such as erasing all the data on a disc.
+
+- Display optical drives avaliable to wodim:
+
+`wodim --devices`
+
+- Display help switches for use with wodim:
+
+`cdrecord`
+
+- Record a pure CD-DA audio:
```

Comment:
Perhaps "Record pure CD-DA audio to a disk" would read better? Either way the "a" doesn't read right.